pkg/controller/plan/adapter/base/doc.go

```diff
@@ -51,6 +52,8 @@ type Builder interface {
 	ResolvePersistentVolumeClaimIdentifier(pvc *core.PersistentVolumeClaim) string
 	// Conversion Pod environment
 	PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env []core.EnvVar, err error)
+	// Build a PersistentVolumeClaim with SourceRef for VolumePopulator
+	PersistentVolumeClaimWithSourceRef(da ovirt.XDiskAttachment, storageName *string, populatorName string, accessModes []core.PersistentVolumeAccessMode, volumeMode *core.PersistentVolumeMode) *core.PersistentVolumeClaim
```

Comment:
because `StorageClassName *string `json:"storageClassName,omitempty" protobuf:"bytes,5,opt,name=storageClassName"`` we can do non-pointer and then use `&` as well..

Reply:
nah, if it's that way on the source you take the value from, that's ok (I wanted to see if we can minimize the length of the arguments list a bit)